test/suites/parathreads/test_tanssi_parathreads.ts

```diff
@@ -154,8 +280,9 @@ describeSuite({
             id: "T05",
             title: "Blocks are being produced on container 2001",
             test: async function () {
-                // Produces 1 block every 2 slots, which is every 24 seconds
-                await sleep(24000);
+                // Produces 1 block every 5 slots, which is every 60 seconds
+                // Give it a bit more time just in case
```

Comment:
This comment was outdated, slot time is defined in orchestrator chain slots, so 5 slots = 30 seconds.